docs/resources/cluster_personality.md

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "vcf_cluster_personality Resource - terraform-provider-vcf"
+subcategory: ""
+description: |-
+  
+---
+
+# vcf_cluster_personality (Resource)
+
+
+
+
```

Comment:
Here we can copy-paste a bit more text from the API reference/Docs on what this resource does.